packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -153,7 +160,9 @@ export const FeedContainer = ({
           )}
           {isV1Search && (
             <span className="flex flex-row flex-1 mt-4">
-              <SearchBarSuggestionList className="hidden tablet:flex overflow-hidden flex-1 mr-3" />
+              {isTabletAbove && (
+                <SearchBarSuggestionList className="hidden tablet:flex overflow-hidden flex-1 mr-3" />
```

Comment:
Why was the CSS only solution not good enough?

Reply:
There are some that require JS evaluation. However keep note that this PR is still WIP. A lot of things here are subject to change in my upcoming commits.